Makefile

```diff
@@ -0,0 +1,37 @@
+R_DYNTRACE := ../R-3.5.0-dyntrace/bin/R
+CHECK_OUTPUT_DIRPATH := /tmp
+
```

Comment:
The first task should be:

```
all: install
```

so the default and `all` do install

Reply:
Done